custom_components/tahoma/climate.py

```diff
@@ -1,18 +1,27 @@
 """Support for TaHoma climate devices."""
+import logging
 
 from homeassistant.components.climate import DOMAIN as CLIMATE
+from homeassistant.helpers import entity_platform
 
 from .climate_aeh import AtlanticElectricalHeater
 from .climate_deh import DimmerExteriorHeating
+from .climate_seh import StatelessExteriorHeating
 from .climate_st import SomfyThermostat
 from .const import DOMAIN
 
+_LOGGER = logging.getLogger(__name__)
+
 TYPE = {
     "AtlanticElectricalHeater": AtlanticElectricalHeater,
     "SomfyThermostat": SomfyThermostat,
     "DimmerExteriorHeating": DimmerExteriorHeating,
+    "StatelessExteriorHeating": StatelessExteriorHeating,
 }
 
+SERVICE_MY = "climate_my"
```

Comment:
```suggestion
SERVICE_CLIMATE_MY_POSITION = "set_climate_my_position"
```

Reminder to eventually align this to #242, if we al agree on a final name.